plugins/backend-app/package.json

```diff
@@ -8,8 +8,9 @@
   "license": "ISC",
   "dependencies": {
     "@dotcom-tool-kit/circleci-heroku": "^2.0.0",
+    "@dotcom-tool-kit/node": "^2.0.0",
     "@dotcom-tool-kit/npm": "^2.0.0",
-    "@dotcom-tool-kit/node": "^2.0.0"
+    "@dotcom-tool-kit/secret-squirrel": "file:../secret-squirrel"
```

Comment:
@ivomurrell i will need your input on how to release this cos i have no idea

Reply:
the best I can come up with is to add the plugin to `backend-app` in a separate PR, sorry. though I'd also suggest that maybe we should be waiting for the `secret-squirrel` plugin to hit 1.0 before adding it to `backend-app` so that it doesn't cause any breaking changes here?